### PR TITLE
fix(components): [tree-v-2] `setExpandedKeys` instead of `setExpendKeys`

### PR DESCRIPTION
**Pull request by holazz**
Please make sure these boxes are checked before submitting your PR, thank you!

- [x] Make sure you follow contributing guide [English](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.en-US.md) | ([中文](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.zh-CN.md) | [Español](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.es.md) | [Français](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.fr-FR.md)).
- [x] Make sure you are merging your commits to `dev` branch.
- [ ] Add some descriptions and refer to relative issues for your PR.

### CODE REVIEW

File: packages/components/tree-v2/src/composables/useTree.ts [hunk 1]
```diff
@@ -191,7 +191,7 @@ export function useTree(props: TreeProps, emit) {
     }
   }
 
-  function setExpendKeys(keys: TreeKey[]) {
+  function setExpandKeys(keys: TreeKey[]) {
```
Comment (plainheart) [inline packages/components/tree-v2/src/composables/useTree.ts:194]: `setExpandedKeys` will be better. `Expand` is a verb.

Comment (holazz) [reply packages/components/tree-v2/src/composables/useTree.ts:194]: Okay!

File: docs/en-US/component/tree-v2.md [hunk 2]
```diff
@@ -105,7 +105,7 @@ tree-v2/filter
 | getCheckedKeys | If the node can be selected (`show-checkbox` is `true`), it returns the currently selected array of node's keys | `(leafOnly: boolean)` |
 | setCheckedKeys | set certain nodes to be checked | `(keys: TreeKey[])` |
 | setChecked | set node to be checked or not | `(key: TreeKey, checked: boolean)` |
-| setExpendKeys | set certain nodes to be expand | `(keys: TreeKey[])` |
+| setExpandedKeys | set certain nodes to be expand | `(keys: TreeKey[])` |
```
Comment (plainheart) [inline docs/en-US/component/tree-v2.md:108]: `to be expand` -> `to be expanded`

Comment (holazz) [reply docs/en-US/component/tree-v2.md:108]: Updated.

**Review by plainheart — APPROVED** [review]
LGTM :)

**Pull request merged**